File: graphql-todo-items-java/src/main/java/com/example/subgraphtemplatejavadgsshows/ApiContainer.java
package com.example.subgraphtemplatejavadgsshows;

import org.openapitools.client.apis.TodoItemControllerApi;
import org.springframework.stereotype.Service;

@Service
public class ApiContainer {
    public TodoItemControllerApi todoItem() {
        return new TodoItemControllerApi();
    }
}

File: graphql-todo-items-java/src/main/java/com/example/subgraphtemplatejavadgsshows/APiContainer.kt
package com.example.subgraphtemplatejavadgsshows

import com.example.subgraphtemplatejavadgsshows.types.TodoItem
import org.openapitools.client.apis.TodoItemControllerApi
import org.springframework.stereotype.Service

@Service
class APiContainer {
    fun todoItem(): TodoItemControllerApi {
        return TodoItemControllerApi()
    }
}
